src/main.cpp

```diff
@@ -583,7 +583,8 @@ int main(int argc, char** argv) {
     // ------- execution -------------
     /* translate AST to RAM */
     debugReport.startSection();
-    Own<ram::TranslationUnit> ramTranslationUnit = AstToRamTranslator().translateUnit(*astTranslationUnit);
+    Own<ram::TranslationUnit> ramTranslationUnit =
+            ast2ram::AstToRamTranslator().translateUnit(*astTranslationUnit);
```

Comment:
Since we're already obfuscating the type, we could also simplify the presentation:
```suggestion
    auto ramTranslationUnit = ast2ram::AstToRamTranslator().translateUnit(*astTranslationUnit);
```